apollo-sample/src/main/java/com/example/apollographql/sample/MainActivity.java

```diff
@@ -23,39 +27,60 @@
     super.onCreate(savedInstanceState);
     setContentView(R.layout.activity_main);
     final TextView txtResponse = (TextView) findViewById(R.id.txt_response);
-    SampleApplication application = (SampleApplication) getApplication();
-    application.githuntApiService()
-        .githuntFeed(new OperationRequest<>(
-            new FeedQuery(FeedQuery.Variables.builder()
-                .limit(10)
-                .type(FeedType.HOT)
-                .build())))
+    final RecyclerView responses = (RecyclerView) findViewById(R.id.responses);
+
+    final SampleApplication application = (SampleApplication) getApplication();
+
+    final Subject<Integer> upvoteObserver = PublishSubject.create();
+
+    final PostsAdapter adapter = new PostsAdapter(upvoteObserver);
+    responses.setAdapter(adapter);
+    responses.setLayoutManager(new LinearLayoutManager(this));
+
+    upvoteObserver.subscribe(new SimpleObserver<Integer>() {
```

Comment:
I guess you should keep subscription and then unsubscribe it in `onDestroy` or even in `onStop`